.github/workflows/build.yml

```diff
@@ -29,8 +29,7 @@ env:
   CURL_CACHE_DIR: ~/.cache/curl
   IMAGE_NAME: cisagov/con-pca-tasks
   PIP_CACHE_DIR: ~/.cache/pip
-  PLATFORMS: "linux/amd64,linux/arm/v6,linux/arm/v7,\
-  linux/arm64,linux/ppc64le,linux/s390x"
+  PLATFORMS: "linux/amd64,linux/arm64"
```

Comment:
If we ever decide to remove platforms, it's best to include a comment with some explanation of why we are doing so.  Can you please add a brief comment here?

Reply:
@dav3r resolved with [06bd355](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/06bd355907f2ed9b9f602b6b3ab27dbb1db394bf)

Reply:
I was thinking of something more along the lines of:
```
# These are the only platforms currently supported by cisagov/con-pca-tasks
```